packages/core/contracts-ovm/insured-bridge/implementation/BridgePool.sol

```diff
@@ -116,15 +116,14 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20 {
         _;
     }
 
-    // TODO: should we consider changing the name of the LP token as a function of the l1Token? if so, might not be able
-    // to do this with this contract inheriting from expanded ERC20 or might need this contract to have an instance
-    // of the LPToken.
     constructor(
+        string memory _lpTokenName,
```

Comment:
nice

Reply:
should we do any validation on these params within the constructor?

Reply:
what did you have in mind? check that theyre not ""?

Reply:
ye I think checking they are not blank is sufficient